test-requirements.txt

```diff
@@ -5,3 +5,5 @@ nose==1.3.7
 pinocchio==0.4.2
 python-dotenv==0.9.1
 asynctest==0.12.3
+pluggy==0.13.1
```

Comment:
err what was this added for?

Reply:
@hampsterx So I had to add that for Tox to make it work in pycharm. If we need to pop that out, that's fine.